### PR TITLE
Hieu tutorials

### PR DESCRIPTION
**Pull request by hieuchi911**

### CODE REVIEW

File: ablator/utils/base.py [hunk 1]
```diff
@@ -241,11 +241,9 @@ def init_weights(module: nn.Module):
 
     Notes
     -----
-    - If the module is a Linear layer, initialize weight values from a normal distribution N(mu=0, std=1.0).
-    If biases are available, initialize them to zeros.
+    - If the module is a Linear layer, initialize weight values from a normal distribution N(mu=0, std=1.0). If biases are available, initialize them to zeros.
```
Comment (fostiropoulos) [inline ablator/utils/base.py:244]: @hieuchi911 in general it is not a good idea to mix PRs. e.g. fixing docs and writing tutorials.

**Review by fostiropoulos — DISMISSED** [review]

File: docs/source/index.rst [hunk 2]
```diff
@@ -16,9 +16,30 @@ Welcome to ablator's documentation!
...
+
+   notebooks/Searchspace-for-diff-optimizers.ipynb
+
 Indices and tables
 ==================
 
 * :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
+ :ref:`search`
```
Comment (fostiropoulos) [inline docs/source/index.rst]: was this added by mistake? It seems to be repeating?

Comment (hieuchi911) [reply docs/source/index.rst]: It's a mistake, fixed in the new commit

File: ablator/utils/base.py (hunk 1 above)
Comment (hieuchi911) [reply ablator/utils/base.py:244]: Do I need to separate this to a new branch?

**Review by fostiropoulos — APPROVED** [review]

**Pull request merged**